fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/ModNioResourcePack.java

```diff
@@ -51,7 +53,7 @@
 import net.fabricmc.loader.api.ModContainer;
 import net.fabricmc.loader.api.metadata.ModMetadata;
 
-public class ModNioResourcePack extends AbstractFileResourcePack implements ModResourcePack {
+public class ModNioResourcePack implements ResourcePack, ModResourcePack {
```

Comment:
Since we're editing this file, could you put an `@ApiStatus.Internal` here?

Reply:
I think the fact that it is in the impl package implies that, but sure